lld/wasm/InputFiles.cpp

```diff
@@ -451,6 +452,18 @@ void SharedFile::parse() {
   }
 }
 
+/// Returns the alignment for a custom section. This is used to concatenate
+/// custom sections with the same name into a single custom section.
```

Comment:
We don't use the /// style comment anywhere else in `wasm/lld`, maybe just a normal `//` comment?